doc/Guildmud Persistance model.md

```diff
@@ -0,0 +1,25 @@
+A simple proposal for a persistance model.
+
+Using @zachflower [PR9](https://github.com/mudcoders/guildmud/pull/9) as a guideline, let's implement data persistance in sqlite.
+
+We have three options to persist the data in sqlite, from simpler to most complex:
+
+1. Give an id to every data structure in the system, and store them in a single table. Content of the data structure is the persisted data structure (persistance model to be defined)
+
+![DB Model](file:///Users/rogersm/Dropbox/dev/guildmud/doc/db_model1.png)
```

Comment:
These don't seem to be pulling up when previewing the file. Feel free to put the images in the repository and reference them from there.

Reply:
Why did this get pushed? This is something that was not ready to be published.

Will cancel the PR and send a new one.